src/test/java/org/cactoos/scalar/FirstOfTest.java

```diff
@@ -79,4 +81,23 @@ public void returnsFallbackIfNothingMatches() {
         );
     }
 
+    @Test
+    public void shouldThrowExceptionForNotFoundElementTest() {
+        new Assertion<>(
+            "Should not return anything",
```

Comment:
@fanifieiev Th is test looks a little confused to me:
- the method signature says that we are testing if we are throwing an exception for element not found, but the message says does not state about that;
- essentially this test is testing the same thing that `returnsFallbackIfNothingMatches`

Reply:
@paulodamaso Right, will fix it